ietf-scitt-charter.md

```diff
@@ -1,35 +1,42 @@
 Introduction
 ============
-Over the years, rapid technological advancements have motivated organizations to implement efficient processes for supply chains (i.e., from design to manufacturing, logistics, and just-in-time delivery).
-While these improvements help organizations increase efficiency and swiftly bring innovations to market, the rapid increase in scale, size, and complexity of supply chains has led to more frequent and sophisticated supply chain attacks.
-The traditional methods of safeguarding supply chains (e.g., pre- and post-audit methodologies) are no longer adequate.
+A Device's supply chain, including its components, is complex often being comprised of both common libraries and custom components. Knowing the identity of a Device or its component is critical to determining its compliance. However, the scale, size, and complexity of supply chains, from design, manufacturing, logistics, and just-in-time delivery has led to more frequent and sophisticated supply chain attacks. The traditional methods of safeguarding supply chains (e.g., pre- and post-audit methodologies) are no longer adequate, therefore, with the increase in suppliers and increase in the complexity of the various supply chains the means to determine a Device or component's identity requires standardization.
+
+[//]: # (MW: The previous introduction sounded like this is a new problem -- it isn't. It's only recently gained importance and recognition due to the increase in interconnected devices. If we are going to cite a rationale to dealing with this now it is the complex supply chain from various commonly used and complexity of component distribution and their diverse sources.)
 
 The output of the SCITT WG is a set of standards that define the essential building blocks enabling the security of supply chain systems and assisting implementers in securing deployments.
 For example, a public computer interface system could report its software composition, which can be compared against known software compositions for such a device, as recorded in a public append-only transparent registry.
 Therefore, providing an individual using the system with confidence that it will behave as and when expected, consistently and without deviation.
 
+[//]: # (MW: "recorded in a public ledger, " I thought that is too implementation specific for an introduction)
```

Comment:
agree

Reply:
Also agree.  Additionally, I suggest a few capitalization and grammatical edits.

SCITT forms a set of building blocks that will allow implemeters to build interoperable supply chain integrity systems. For example, a public computer interface system could report its software composition that can be compared against known software compositions for such a device providing an individual using the system confidence that it will behave as expected, when expected, and nothing/nowhen else

Reply:
resolved in previous commits and reflected in current branch. also removed "public" as that can be misunderstood easily

Reply:
@kaywilliams I like where that is going. May I offer this slight mod. I'm concerned that we're swerving a little into the 'preventative security' world where actually the work here is transparency and accountability. In the example given we don't have total confidence that the computer will always behave perfectly: there may be bugs or operator-borne attacks, for example. But what we do have is confidence that you're dealing with the computer system you thought you would be, and that nothing has been injected or modified on its way to you. 

I also removed "an individual" because humans aren't realistically going to be doing much of this transactional verification: we want all of this stuff to be strong enough that we can automate all the mundane validation work away.

_"SCITT forms a set of interoperable building blocks that will allow implementers to build integrity and accountability into supply chain systems. For example, a public computer interface system could report its software composition that can then be compared against known software compositions (and certifications?) for such a device thereby giving confidence that the system is running the software expected and has not been modified, either by attack or accident, in the supply chain"._